File: lista2sala/9.py
n1=int(input("digite o 1º número inteiro:"))
n2=int(input("digite o 2º número inteiro:"))
if n1%4==0==0 or n1%5==0 or n2%2==0 or n2%3==0:
    if n1%4==0 or n1%5==0:
        print("numero divisivel por 4 ou 5:",n1 )
    if n2%4==0 or n2%5==0:
        print("numero divisivel por 4 ou 5:", n2)
else:
    print("não foi digitado nenhum número divisivel por 4 ou 5")

File: lista2sala/9.c
#include <stdio.h>
int main (){
    int n1, n2;
    printf("digite os números:");
    scanf("%d%d", &n1, &n2);
    if ((n1%4==0)||(n1%5==0)||(n2%4==0) || (n2%5==0)){
        if ((n1%4==0) && (n1%5==0)){
            printf("numero divisivel por 4 ou 5:  %d", n1);
        }
        if ((n2%4==0) && (n2%5==0)){
            printf("numero divisivel por 4 ou 5:  %d", n2);
        }
    }
    else{
        printf("print não foi digitado nenhum numero divisivel por 4 ou 5:");
    }
}